android/src/main/java/com/swmansion/rnscreens/RNScreensPackage.kt

```diff
@@ -18,4 +23,31 @@ class RNScreensPackage : ReactPackage {
             ScreenStackHeaderSubviewManager(),
             SearchBarManager()
         )
+
+    override fun getModule(
+        s: String,
+        reactApplicationContext: ReactApplicationContext
+    ): NativeModule? {
+        when (s) {
+            ScreensModule.NAME -> return ScreensModule(reactApplicationContext)
+        }
+        return null
+    }
+
+    override fun getReactModuleInfoProvider(): ReactModuleInfoProvider {
+        return ReactModuleInfoProvider {
+            val moduleInfos: MutableMap<String, ReactModuleInfo> = HashMap()
+            val isTurboModule = BuildConfig.IS_NEW_ARCHITECTURE_ENABLED
+            moduleInfos[ScreensModule.NAME] = ReactModuleInfo(
+                ScreensModule.NAME,
+                ScreensModule.NAME,
+                false, // canOverrideExistingModule
+                false, // needsEagerInit
+                true, // hasConstants
+                false, // isCxxModule
+                true // isTurboModule
```

Comment:
Shouldn't we use `isTurboModule` value here, instead of straight up passing `true`? It this is not the case `isTurboModule` is redundant.